GVFS/GVFS.Virtualization/FileSystem/FileSystemVirtualizer.cs

```diff
@@ -16,7 +16,10 @@ public abstract class FileSystemVirtualizer : IDisposable
         protected static readonly byte[] FolderContentId = Encoding.Unicode.GetBytes(GVFSConstants.AllZeroSha);
 
         protected static readonly GitCommandLineParser.Verbs CanCreatePlaceholderVerbs =
-            GitCommandLineParser.Verbs.AddOrStage | GitCommandLineParser.Verbs.Move | GitCommandLineParser.Verbs.Status;
+            GitCommandLineParser.Verbs.AddOrStage |
+            GitCommandLineParser.Verbs.Move |
+            GitCommandLineParser.Verbs.Status |
+            GitCommandLineParser.Verbs.CheckIgnore;

```

Comment:
Do we always want to allow `check-ignore`, or only when it has a `--stdin` parameter? What about other verbs that take a `--stdin` parameter, could they be in a similar boat?

Reply:
Commands that could be waiting on stdin
```
check-attr
check-ignore
check-mailmap
checkout-index
commit-graph
diff-tree
fetch-pack
hash-object
index-pack
name-rev
notes
reset  -  Experimental
send-pack
rev-list
update-index
update-ref
```

The fact that these wait on stdin isn't necessarily an issue, but placeholders will not be able to be created if these are running and waiting on input.

Reply:
Running some tests and the failures are for:
```
check-ignore
check-mailmap
checkout-index
reset
update-index
```

Most of the others are in the list to not get the lock which is probably where `check-ignore` and `check-mailmap` belong.  I'll update the PR.  The others will have to stay for now since they are changing the index.